packages/react-core/src/components/Alert/Alert.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import { StyleSheet, css, getModifier } from '@patternfly/react-styles';
+import PropTypes from 'prop-types';
+import styles from '@patternfly/patternfly-next/components/Alert/styles.css';
+
+import AlertIcon from './AlertIcon';
+import AlertBody from './AlertBody';
+import AlertAction from './AlertAction';
+import { capitalize } from '../../internal/util';
+
+const utils = {
+  ...StyleSheet.create({
```

Comment:
this spread is not needed, but fine to leave.

Reply:
Yeah, I wanted to join `styles.utils` and these utils together, but babel kept complaining that `utils` does exists on styles object so I removed it and kept the spread there. It's not needed, so I remove it and assign it directly to `utils` constnat.